### ISSUE TITLE
Color column stays highlighted after deleting text (full_column = true)

### ISSUE THREAD

**Issue by jessevdp** [fmbarina/multicolumn.nvim#1]
Once I add text that protrudes the ruler I set it becomes visible.
<img width="831" alt="Screenshot 2024-07-01 at 14 42 24" src="https://github.com/fmbarina/multicolumn.nvim/assets/15139826/54491177-b2bc-44a1-8d8c-acdb4560fb3f">

When I then delete the text the column stays highlighted whatever I do.
<img width="792" alt="Screenshot 2024-07-01 at 14 42 44" src="https://github.com/fmbarina/multicolumn.nvim/assets/15139826/60f2d76a-58b9-430e-b5e6-a83295192737">

When I quit and re-open nvim the ruler disappears.
<img width="1030" alt="Screenshot 2024-07-01 at 14 43 29" src="https://github.com/fmbarina/multicolumn.nvim/assets/15139826/1a8b9905-995b-4366-a986-d934428c4ed2">

My config:
```lua
return {
  'fmbarina/multicolumn.nvim',
  event = { 'BufReadPre', 'BufNewFile' },
  opts = {
    base_set = {
      full_column = true,
      always_on = false,
    },
  },
}
```

### What I expected

Shouldn't the ruler disappear? Since I have `always_on` disabled.

**Comment by fmbarina** [comment]
You're right, it should. Thank you for reporting. Haven't used Neovim in a while, so it's possible the plugin has fallen behind on necessary updates. I'm picking it back up, so I'll look into it.

**Comment by jessevdp** [comment]
Awesome.

I really appreciated the ability to have _**multiple**_ columns. I don't think other plugins have that. So thanks for that 😄

**Comment by fmbarina** [comment]
I'm glad to help, but I'm not sure what you mean by multiple columns here. [Smartcolumn](https://github.com/m4xshen/smartcolumn.nvim) supports per-filetype, multiple columns and, from a quick glance at [NeoColumn](https://github.com/ecthelionvi/NeoColumn.nvim/blob/main/lua/NeoColumn.lua#L112)'s code, I think it supports multiple columns, too.

**Comment by jessevdp** [comment]
I guess I must have misunderstood the differences then. (After some short searching for a colorcolumn plugin I settled on this one. Misinformation might be from reddit or something.)

**Comment by fmbarina** [comment]
Hey again, finally back. It was a very silly bug :^)

Despite the docs having long stated that the default value for the colorcolumn option is `""` (an empty string), multicolumn cleared it by setting the value to `nil`. I probably copied this from someone else without properly understanding it. Something happened, I'm guessing in the update to 0.10, that made it so this no longer worked. Changing it to `""`, as it should've been from the start, did the trick.

Perhaps you've switched to another plugin at this point, but regardless, thanks again for reporting :^)

I'm marking this issue as closed in 5b30a04.

**Comment by jessevdp** [comment]
Awesome! Thanks 😄
